src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -233,6 +241,72 @@ const ZUIOrganizeSidebar = (): JSX.Element => {
             })}
           </List>
         </Box>
+        <Box
+          sx={{
+            bottom: 0,
+            position: 'absolute',
```

Comment:
This is not ideal, because absolute positioning takes the element out of the normal column flow, and that will probably make it difficult to create the expanded organization switcher. But that problem can be left to whomever takes on that task (probably you?). 😊